core/database/tag.go

```diff
@@ -0,0 +1,60 @@
+package database
+
+import (
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/dynamodb"
+	"github.com/aws/aws-sdk-go/service/dynamodb/dynamodbattribute"
+	"github.com/ubclaunchpad/pinpoint/core/model"
+)
+
+// AddNewTag adds a tag to the database
+func (db *Database) AddNewTag(t *model.Tag, c *model.Club) error {
+	input := &dynamodb.PutItemInput{
+		Item: map[string]*dynamodb.AttributeValue{
+			"pk":       {S: aws.String(t.ApplicantID)},
+			"sk":       {S: aws.String(t.PeriodEventID)},
+			"tag_name": {S: aws.String(t.TagName)},
+			"type":     {S: aws.String("tag")},
+		},
+		TableName: aws.String("ClubData-" + c.ID),
+	}
+	if _, err := db.c.PutItem(input); err != nil {
+		return err
+	}
+	return nil
+}
+
+// GetTag gets the tag associated with Applicant_ID, Period_ID, & Event_ID
+func (db *Database) GetTag(ApplicantID string, PeriodID string, EventID string, c *model.Club) (*model.Tag, error) {
```

Comment:
in Go, variable names should be lowercased, as uppercased names have a special meaning in a lot of contexts (exports)

```suggestion
func (db *Database) GetTag(applicantID string, periodID string, eventID string, c *model.Club) (*model.Tag, error) {
```

Reply:
fixed it!